python/cudf/cudf/bindings/rolling.pyx

```diff
@@ -68,16 +79,16 @@ def apply_rolling(inp, window, min_periods, center, op):
                 c_min_periods_col,
                 c_forward_window_col
             )
-        data, mask = gdf_column_to_column_mem(output_col)
-
-    result = Column.from_mem_views(data, mask)
+        # I'd expect this to work but it doesn't...
+        # result = gdf_column_to_column(c_output_col)
```

Comment:
Figuring this out was weird. I couldn't figure out what's different about this line vs. the two below it, since this function contains those two lines.

Reply:
I think lets just open an issue and investigate post release, only thing I can think of is something weird with Cython scoping of Python objects between cdefs